package/upgrade/upgrade_manifests.sh

```diff
@@ -1205,8 +1205,12 @@ upgrade_addon_rancher_logging()
 {
   echo "upgrade addon rancher_logging"
   # .spec.valuesContent has dynamic fields, cannot merge simply, review in each release
-  # in v1.3.0, patch version is OK
-  upgrade_addon_try_patch_version_only "rancher-logging" "cattle-logging-system" $REPO_LOGGING_CHART_VERSION
+  # in v1.3.0, the fluentbit image needs to be patched
+  if [ "$REPO_LOGGING_CHART_VERSION" = "103.0.0+up3.17.10" ]; then
```

Comment:
Is there a possiblity this condition would be met?

Reply:
In v1.3.0, we bumped the rancher-logging chart to `103.0.0+up3.17.10`, and hacked the chart to use fluent-bit new image for CVE fix. 

The `REPO_LOGGING_CHART_VERSION` is taken from the upgrade target ISO. It means as long as the new ISO (any version 1.3.0 local/formal release after this PR) is using `103.0.0+up3.17.10`, the script will try to amend the old fluentbit 1.9.5 image tag. 

v1.2.1-> v1.3.0(any), will go through the new function and amend the addon valuesContent;
v1.3.0(any) -> v1.3.0(any), will also go through the new function, but when fluentbit 1.9.5 image tag is not found in the valuesContent, it fallback to the normal upgrade.